tests/TestGalaxyObjects.py

```diff
@@ -617,6 +627,28 @@ def test_update(self):
         self.assertEqual(self.hist.annotation, new_annotation)
         self.assertEqual(self.hist.tags, new_tags)
 
+    def test_new_dataset_collection(self):
+        collection_description = self._create_collection_description()
+        dataset_collection = self.hist.new_dataset_collection(collection_description)
+        self.__check_dataset_collection(dataset_collection)
+
+    def test_delete_dataset_collection(self):
+        collection_description = self._create_collection_description()
+        dataset_collection = self.hist.new_dataset_collection(collection_description)
+        dataset_collection.delete()
+        self.assertTrue(dataset_collection.deleted)
+
+    def _create_collection_description(self):
+        dataset1 = self.hist.paste_content(FOO_DATA)
+        dataset2 = self.hist.paste_content(FOO_DATA_2)
+        collection_description = collections.CollectionDescription(
+            name="MyDatasetList",
+            elements=[
+                collections.HistoryDatasetElement(name="sample1", id=dataset1.id),
+                collections.HistoryDatasetElement(name="sample2", id=dataset2.id),
+            ]
+        )
+        return collection_description
 
```

Comment:
You need to add one more blank line here, flake8 is complaining:

./tests/TestGalaxyObjects.py:653:1: E302 expected 2 blank lines, found 1